documentation/modules/configuring-mtv-operator.adoc

```diff
@@ -60,12 +61,31 @@ spec:
 |`10`
 
 |`controller_filesystem_overhead`
-|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`. Note, this setting can only be changed using the CLI.
+|Percentage of space in persistent volumes allocated as file system overhead when the `storageclass` is `filesystem`.
 |`10`
 
 |`controller_block_overhead`
-|Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk. Note, this setting can only be changed using the CLI.
+|Fixed amount of additional space allocated in persistent block volumes. This setting is applicable for any `storageclass` that is block-based. It can be used when data, such as encryption headers, is written to the persistent volumes in addition to the content of the virtual disk.
+|`ForkliftController` CR and *Settings* of the *Overview* page.
```

Comment:
also this one and the ones below should be `ForkliftController CR only` (`ForkliftController CR and *Settings* of the *Overview* page` applies to the first entries but you wrote above that this the default unless written otherwise)